### DISCUSSION TITLE
How can I make fields in the Forms optional instead of required?

### DISCUSSION THREAD

**Discussion by IcreatedThisForMyGame** [statamic/cms#7236] [Troubleshooting]
I am editing the YAML file that looks like this: 

ections:
  main:
    display: Main
    fields:
      -
        handle: company
        field:
          input_type: text
          antlers: false
          display: Unternehmen
          type: text
          icon: text
          listable: hidden
          validate:
            - required
      -
        handle: employees
        field:
          display: Mitarbeiterzahl
          type: text
          icon: text
          listable: hidden
          validate:
            - required
...... 

I want the fields to be able to be left empty / optional. I tried to remove "validate: -required", I removed the "required", I changed "required" to "optional" -> 500 server error :/

**Also another question** ; If I want to expand the form I also get 500 Server errror. Am I correct that I would just have to copy one of the secitons in this YAML file and also copy a HTML section under resources/views/partials/shared/form.antlers.html? Or are there other files that also need to be edited to make a form have more fields? thanks.

**Comment by duncanmcclean** [comment]
Just remove the `validate` section, then it'll be optional.

```
validate:
- required
```

**Reply by IcreatedThisForMyGame** [reply]
Thank you for the Answer. I already tried that but I just found the issue: There was another "required" in the HTML section: 

            <div class="sm:col-span-2">
                <label for="email-address" class="block text-sm font-medium text-white">E-Mail-Adresse *</label>
                <div class="mt-1">
                    <input
                        class="py-3 px-4 block w-full shadow-sm focus:ring-bavarian-blue focus:border-bavarian-blue border-gray-300 rounded-md"
                        type="email" name="email-address" required>
                </div>
            </div>

So removing that required fixed it. I was looking in the wrong place. Thanks for your time.

**Reply by duncanmcclean** [reply]
No worries - have a great rest of your day!
